src/main/java/org/jenkinsci/plugins/pitmutation/portlets/CoverageRange.java

```diff
@@ -11,13 +11,14 @@
      * last
      */
     PERFECT(100.0, new Color(0x00, 0x8B, 0x00), new Color(0xEE, 0xEE, 0xEE)), //
-    EXCELLENT(97.0, new Color(0x00, 0xCD, 0x00), new Color(0x00, 0x00, 0x00)), //
-    GOOD(92.0, new Color(0x7A, 0xFF, 0x3F), new Color(0x00, 0x00, 0x00)), //
-    SUFFICIENT(85.0, new Color(0xC8, 0xFF, 0x3F), new Color(0x00, 0x00, 0x00)), //
-    FAIR(75.0, new Color(0xFF, 0xFF, 0x00), new Color(0x00, 0x00, 0x00)), //
-    POOR(50.0, new Color(0xFF, 0x7F, 0x00), new Color(0x00, 0x00, 0x00)), //
-    TRAGIC(25.0, new Color(0xFF, 0x00, 0x00), new Color(0xEE, 0xEE, 0xEE)), //
-    ABYSMAL(0.0, new Color(0x00, 0x00, 0x00), new Color(0xEE, 0xEE, 0xEE)),
+    EXCELLENT(90.0, new Color(0x00, 0xCD, 0x00), new Color(0x00, 0x00, 0x00)), //
```

Comment:
What was the reasoning behind changing this? I'm not against it, just wondering if there was a reason.

Reply:
I changed the thresholds and added some extra options so that the package/class/module view have a more sliding scale of colors. I also had to change some of the font colors so they would be readable on the UI.

Reply:
Since we are currently changing the default colors in Jenkins it might make sense to replace that in the future by a class and use the colors from the CSS files (see https://docs.google.com/presentation/d/1ulLO46YPDvleL1jGxRDu8ah598rC69gJt2wJ3G5CYVg/edit#slide=id.g8aa9d1f643_0_270 for details). Then the colors would also work in a dark theme. If you are interested in refining the requirements: we are meeting every 2 weeks in the [UX SIG](https://www.jenkins.io/sigs/ux/) and discuss such topics. Maybe it would make sense to define a Jenkins wide public color palette that can be used by static analysis tools (my warnings plugin), the code coverage API plugin and the PIT plugin.

Reply:
Happy to come along and contribute @uhafner. This is my first major contribution, but I'm happy to help out :) When is the next meeting?

Just out of curiosity, is this repo regularly maintained and used? When we installed the plugin, it wasn't in a particularly great shape, so just wondering if that's something to do with our specific Jenkins version or whether it's just not that widely used.

Reply:
The next meeting is this week: https://www.jenkins.io/events/

Reply:
@uhafner Ahh sorry, I have meetings at that time. I'm happy to contribute and make any changes as required though. can catch up separately if there's some stuff that needs doing to this repo?